docs/getting-started/github-apps/github-apps.rst

```diff
@@ -59,16 +48,16 @@ With the Dockstore GitHub App installed, the synchronization is done automatical
 you add a new branch or release of a workflow on GitHub, Dockstore is notified,
 and Dockstore updates its copy of the workflow. For example, After publishing a new release
 of a workflow on GitHub, a new version of the workflow will be present in
-Dockstore shortly afterwards.
+Dockstore shortly afterwards. For this to work, a .dockstore.yml file is **required** in the root directory (or inside the ``.github`` directory) of each GitHub repository you want
+to associate with a workflow/tool/service on Dockstore. It should not be inside any subfolder (except ``.github``).
 
-For this to work, a ``/.dockstore.yml`` file is **required in the root directory of each GitHub repository** you want
-to associate with a workflow on Dockstore. A template for both workflows and services are shown below,
-along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
+Simple templates for tools, workflows, and services are shown below,
+as well as links to more advanced explanations of every field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
 Error Handling
 ----------------------------------
-It is possible for an invalid ``/.dockstore.yml`` to cause an errors. If we cannot read the 
+It is possible for an invalid .dockstore.yml to cause an errors. If we cannot read the 
```

Comment:
`It is possible for an invalid .dockstore.yml to cause an errors` -- I would think it would always cause errors.